elliottlib/cli/__main__.py

```diff
@@ -360,8 +376,14 @@ def repair_bugs(runtime, advisory, auto, id, original_state, new_state, noop, de
     green_print("Got bugs for advisory")
     for bug in attached_bugs:
         if bug.status in original_state:
+            if close_placeholder and "Placeholder" in bug.summary:
+                elliottlib.bzutil.set_state(bug, "CLOSED", noop=noop)
+            else:
+                elliottlib.bzutil.set_state(bug, new_state, noop=noop)
+                # only add comments for non-placeholder bug
+                if comment and not noop:
+                    bug.setstatus(comment=comment, private=False)
```

Comment:
The `else` branch will be reached if  `close_placeholder == False` or `"Placeholder" not in bug.summary`.